config/settings.yml

```diff
@@ -53,6 +53,8 @@ api_timeout: 300
 web_timeout: 30
 # Periods (in hours) which are allowed for user blocks
 user_block_periods: [0, 1, 3, 6, 12, 24, 48, 96, 168, 336, 731, 4383, 8766, 87660]
+# Account deletion cooldown period since last changeset close (in hours)
+user_account_deletion_delay: 24
```

Comment:
Do we want to have a default for this? or should it default to off but be something we can enable operationally? In other words should we assume that other users of this code will want this.

Reply:
Let's default to the previous behavior, `user_account_deletion_delay = 0`